aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -350,7 +350,7 @@ When publishing ASP.NET Core web apps, the build artifacts and contents of the *
 
 The preceding markup can be added to a publish profile or the *.csproj* file. When added to the *.csproj* file, the rule is added to all publish profiles in the project.
 
-The following `<MsDeploySkipRules>` element excludes all files from the *wwwroot/content* folder:
+The following `<MsDeploySkipRules>` element excludes all files from the *wwwroot\content* folder:
```

Comment:
Consistency nit ... Line 343 goes with forward slash. I put it in the suggestion for Line 343.